MediaBrowser.Providers/MediaInfo/AudioResolver.cs

```diff
@@ -0,0 +1,146 @@
+#pragma warning disable CA1002, CS1591
+
+using System;
```

Comment:
New files shouldn't need to disable warnings.

```suggestion
using System;
```